posts/2020/07/writing-docs-is-not-just-writing-docs.md

```diff
@@ -0,0 +1,39 @@
+<!--
+.. title: Writing docs is not just writing docs
+.. slug: writing-docs-is-not-just-writing-docs
+.. date: 2020-07-05 12:00:00 UTC-05:00
```

Comment:
Can you update this to the current date once the new theme is deployed and this post will be published?

Reply:
```suggestion
.. date: 2020-07-07 18:00:00 UTC-05:00
```